assets/js/base/components/checkbox-control/style.scss

```diff
@@ -7,21 +7,21 @@
 
 	.wc-block-components-checkbox__input[type="checkbox"] {
 		appearance: none;
-		border: 2px solid currentColor;
+		border: 2px solid #50575e;
```

Comment:
Text inputs use a `#8d96a0` color for the border. Do you think we could use one or the other for both components or is it intended to use different colors for them?